### PR TITLE
[MIRROR] Prevents people from joining with the same character

### PR DESCRIPTION
**Pull request by VirgoBot**
Original PR: https://github.com/PolarisSS13/Polaris/pull/5962
--------------------
Just a check on late spawn to prevent people from rejoining constantly as the same char.

image

### CODE REVIEW

**Comment by Wickedtemp** [comment]
Full disclosure: 

If you tram, cryo, or quit-round, this will not prevent you from respawning as the same character.

**Comment by Iroquois-Pliskin** [comment]
This seems like a really specific problem. I'm curious what caused it.

**Comment by Wickedtemp** [comment]
Probably... people respawning as the same character over and over again?

**Comment by SpadesNeil** [comment]
This sounds like it would be a problem for us.

**Comment by Screemonster** [comment]
On the one hand I can see how this might help with impersonation shit, on the other it'd stop people rejoining when they can't be resleeved.
My gut says the solution is gonna cause more issues for us than the problem, in this case.

**Comment by Wickedtemp** [comment]
It wouldn't stop them from rejoining. All they'd have to do is hit quit-round first.

**Comment by Novacat** [comment]
QTR prevents that character from being used again in that round. While this change might be good with a change to QTR that enables round-rejoining, this change by itself would probably do more harm than good.

That particular idea I like as it would encourage people to use QTR when they can not/do not want to be resleeved.

**Comment by Wickedtemp** [comment]
Oh, it does? I never had to use quit-round, Tempest doesn't ever die in scenes and is always implanted if something bad goes down, so it never came up. In that case... Might be best not to merge, but I'll be honest, if you manage to die without getting an implant first and nobody even knows where your body is, if there is one to be found... That's kind of on you unless there's just literally no one in Medical.

**Comment by Novacat** [comment]
I have done some research, I believe I can remove the no respawns for people who use the QTR verb. That would make this PR acceptable, I believe.

**Comment by SpadesNeil** [comment]
I agree. Be sure to include a message in this prompt that the previous character had not quit the round, or else nobody will figure it out.

**Comment by Very-Soft** [comment]
At present if you QTR as a character you can't respawn as them anyway iirc. I've had to manually spawn in a few people over it, unless that's been changed in the latest update.

**Comment by Novacat** [comment]
The PR has been changed so that QTR no longer prevents you from respawning.

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -169,6 +169,13 @@
 		ViewManifest()
 
 	if(href_list["SelectedJob"])
+	
+		//Prevents people rejoining as same character.
```
Comment (kevinz000) [inline code/modules/mob/new_player/new_player.dm]: i know it isn't virgo's fault but jesus christ reading this snippet of code has hurt me inside

**Pull request merged**